config/locales/activerecord.yml

```diff
@@ -1,5 +1,6 @@
 en:
   errors:
+    # format: "%{message}" #Add this once all other errors updated
```

Comment:
@adrianclay @cesidio I don't suppose you know of a way to achieve the same effect without changing the global error format?

Reply:
@dushan-madetech we did some hacky stuff in the haml files before to avoid this. Since we want to update most of the error messages to a more descriptive instruction, I think changing the global formatting makes more sense

Reply:
> @adrianclay @cesidio I don't suppose you know of a way to achieve the same effect without changing the global error format?

Having developed several governmental websites, I always did that. If you have a look at how GOV.UK design system dictates how error message should be written:

https://design-system.service.gov.uk/components/error-message

you realise that you definitely don't want the standard rails format:
```
format: "%{attribute} %{message}"
```